src/goPackages.ts

```diff
@@ -29,6 +40,11 @@ function gopkgs(): Promise<Map<string, string>> {
 				return promptForMissingTool('gopkgs');
 			}
 
+			if (errchunks && errchunks.join('').startsWith('flag provided but not defined: -workDir')) {
+				promptForUpdatingTool('gopkgs');
+				return Promise.resolve(pkgs);
```

Comment:
fallback to not using `workdir` by `return gopkgs().then(result => resolve(result))` instead.

Reply:
Ahh... good idea

Reply:
Done